### ISSUE TITLE
Wine support?

### ISSUE THREAD

**Issue by YellowOnion** [jackaudio/jack-router#10]
How possible would it be to use this with Wine, and pass the ASIO to Linux jackd/pipewire?

Maybe the linux variant can be "exposed" via winelib?

**Comment by falkTX** [comment]
No, because this is a Windows driver which is only possible to build with MSVC. Neither winegcc or mingw can build it as far as I know.

**Comment by YellowOnion** [comment]
WineASIO compiles with winegcc, and loads in the same way using regsvr32
